autotest/tp-libvirt
# add subcase vcpu_placement to test case vcpupin

test vcpupin feature placement as follows
    1. set vcpu placement to auto and check xml result
    2. set vcpu cpuset without placement defined and check xml result

Signed-off-by: Jin Li <jil@redhat.com>
